### DISCUSSION TITLE
Can't Connect the Mobile Phone when Using USB Type C socket

### DISCUSSION THREAD

**Discussion by brucetheelephant** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#630] [Q&A] [answered]
Hello!

I'm not sure if I'm allowed to ask this here. If not, I apologize and.. maybe someone can suggest an alternative? Thank you!

I have two mobile devices, a (heavily debloatted) Xiaomi mobile phone and a (lightly debloatted (for now)) Samsung tablet.

The Xiaomi mobile phone does not connect to the PC when using a USB Type C socket, it just charges. It does connect when using a USB Type A socket. The situation is the same in Linux Plasma and Windows 10/11.

For the record this is the first time I'm trying to connect this phone to a PC using a USB Type C socket.

The Samsung tablet connects under all circumstances.

Maybe I removed something which I shouldn't have!? I have checked, and double-checked and triple-checked but I just can't figure out if I did something wrong.

This is the Apps Backup List for the Xiaomi mobile phone: [2024-09-16_15-27-53.json](https://github.com/user-attachments/files/17018219/2024-09-16_15-27-53.json).

Can someone help? Any help is welcome!

Thank you!

**Comment by lavafroth** [comment]
It's probably an issue with the USB cable itself, can speak from experience.

**Comment by AnonymousWP** [comment]
Yeah, or try another port.

**Comment by brucetheelephant** [comment, marked as answer]
I tried two different cables and both USB Type C sockets. The tablet connects.

I have just did a factory reset on the phone and it still doesn't connect. It's probably a hardware thing.

Thank you for the suggestions though! : )

**Reply by AnonymousWP** [reply]
Does it charge though? Does moving around files between tablet and PC work fine?

**Reply by brucetheelephant** [reply]
The phone charges but it does not show the usual connection pop-up when I connect it to the laptop. I cannot move files around since it cannot connect properly. But, again, this is only when I connect the phone to the laptop USB Type C sockets, all works as it should when I connect it to the USB Type A sockets. It's might strange.

**Reply by AnonymousWP** [reply]
Hmm alright. Sorry, but then I'll close your Discussion that you opened as it's no longer relevant to UAD-ng anymore. Good luck with getting it fixed though!

**Reply by brucetheelephant** [reply]
Yes, please close it! Thank you!

And, I apologize for posting this here! I thought the problem might have been a missing package but I was obviously wrong.
